ROADMAP.md

```diff
@@ -0,0 +1,95 @@
+
+This document presents the roadmap of Ratify that translates our strategy into practical steps.
+
+## Milestones
+
+The Ratify roadmap is divided into milestones, each with a set of features (high level) and timeline. The milestones marked as `Tentative` are subject to change based on the project’s priorities and the community’s feedback. We will prioritize releases for security or urgent fixes, so the roadmap may be adjusted and new features may be postponed to the next milestone. Any dates and features listed below in a given milestone are subject to change. See the [GitHub milestones](https://github.com/deislabs/ratify/milestones?state=open) for the most up-to-date issues and their status. We are targeting to release a new Ratify version every 3 or 4 months.
+
+### v1.0
+
+**Status**: Completed
+
+**Released date**: Sep 27, 2023
+
+**Release link**: [v1.0.0 Release Notes](https://github.com/deislabs/ratify/releases/tag/v1.0.0)
+
+**Major features**
+
+- Ratify as an external Data Provider for Gatekeeper
+- Plugin framework for extensibility
+- Policies for Notary Project signatures verification at admission control in kubernetes
+- Policies for Cosign keyless verification at admission control in kubernetes
+- High Availability support in Kubernetes (Experimental)
+
+### v1.1
+
+**Status**: Completed
+
+**Release date**: Dec 12, 2023
+
+**Release link**: [v1.1.0 Release Notes](https://github.com/deislabs/ratify/releases/tag/v1.1.0)
+
+**Major features**
+
+- Policies for assessing vulnerability reports at admission control in kubernetes
+- Policies for assessing software license at admission control in kubernetes
+- New diagnostic logs
+
+### v1.2
+
+**Status**: In progress
+
+**Target date**: Apr 30, 2024
+
+**major features**
+
+- Kubernetes multi-tenancy support (Namespace-specific policies)
+- OCI v1.1 compliance
+- Cosign signatures verification using keys in AKV
+- Error logs improvements
+
+See details in [GitHub milestone v1.2.0](https://github.com/deislabs/ratify/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.2.0).
+
+### v1.3
+
+**Status**: Not started
+
+**Target date**: Jun 30, 2024
+
+**Major features**
+
+- Kubernetes multi-tenancy support (Verifying Common images across namespaces)
+- Cosign keyless verification using OIDC settings
+- Notary Project signature verification with Time-stamping support
+- Signing Certificate/key rotation support
+
+See details in [GitHub milestone v1.3.0](https://github.com/deislabs/ratify/issues?q=is%3Aopen+is%3Aissue+milestone%3Av1.3.0).
+
+### v1.4
+
+**Status**: Tentative
+
+**Target date**: Sep 30, 2024
+
+**Major features**
+
+- Attestations support
+- Use Ratify at container runtime (Preview)
```

Comment:
This is not likely happening by September.

Reply:
Thanks @luisdlp. We can update this file in next iteration.